src/Components/Timer/Timer.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react'
+import './Timer.scss'
+
+interface Props {
+  total_seconds: number,
+  stopTimer: (arg0: string) => {}
+}
+
+const Timer: React.FC<Props> = ({ total_seconds, stopTimer }) => {
+  const [ elapsedTime, setElapsedTime ] = useState(0)
+  const [ inProgress, toggleInProgress ] = useState(false)
+  let colorClass: string = 'green'
```

Comment:
When you're assigning the colorClass here, is this being used for CSS too? Really curious as to how the colorClass works!

Reply:
Yes, it will be used in the CSS, though I have not written any of that yet.